Message-Id: <790f6e636b6caf5fa02e645ea0bc473892a0b86e.1336068799.git.agx@sigxcpu.org>
In-Reply-To: <bed21802c06c687175cf38a4a62f586a9945d804.1336068799.git.agx@sigxcpu.org>
References: <5909887366833c92178e3ed650b87d666f9f50c1.1336068799.git.agx@sigxcpu.org>
	<56db9d711b7615ece3848c57f71f362f0e5f4fea.1336068799.git.agx@sigxcpu.org>
	<727a1c60fdc68e24b2f457f5b7e3611b2aae7a7b.1336068799.git.agx@sigxcpu.org>
	<af426bb880140c2941366a3a84fd11a714b9c778.1336068799.git.agx@sigxcpu.org>
	<64360f6da8f2d82618e0eda5d971ee6a083851ae.1336068799.git.agx@sigxcpu.org>
	<308046ae1d09cc7cce80fdb268cfb7fee35de13d.1336068799.git.agx@sigxcpu.org>
	<f96b7f473f61a8e6d4866a49d9fcc861bed443b5.1336068799.git.agx@sigxcpu.org>
	<7e773c9b5d95a42321be6b5fc7a11cf0fd51a124.1336068799.git.agx@sigxcpu.org>
	<6d5763889d5f114ed6f149ecd47c562f06868519.1336068799.git.agx@sigxcpu.org>
	<e82f2035cd9341f3530199e68c11a27790f041de.1336068799.git.agx@sigxcpu.org>
	<87e0b605aa093d95c507083de601261098cd1aff.1336068799.git.agx@sigxcpu.org>
	<d7d5684690a5110e57b4f5a5ba48b76aff0f183c.1336068799.git.agx@sigxcpu.org>
	<574c21aa539700de7e6214f0123644312d1724b3.1336068799.git.agx@sigxcpu.org>
	<414a2a09a0c8f8b00c5804b2d5afd2ae704fd96f.1336068799.git.agx@sigxcpu.org>
	<bed21802c06c687175cf38a4a62f586a9945d804.1336068799.git.agx@sigxcpu.org>
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 2 Apr 2011 10:37:16 +0200
Subject: Brand ru

---
 ru/locale/ru/lightning/lightning.dtd        |    2 +-
 ru/locale/ru/lightning/lightning.properties |    6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/ru/locale/ru/lightning/lightning.dtd b/ru/locale/ru/lightning/lightning.dtd
index c6f5749..d830727 100644
--- a/ru/locale/ru/lightning/lightning.dtd
+++ b/ru/locale/ru/lightning/lightning.dtd
@@ -46,7 +46,7 @@
   -->
 
 <!-- Tools menu -->
-<!ENTITY lightning.taskLabel "Lightning">
+<!ENTITY lightning.taskLabel "Iceowl">
 
 <!-- New menu popup in File menu -->
 <!ENTITY lightning.menupopup.new.event.label    "Событие…">
diff --git a/ru/locale/ru/lightning/lightning.properties b/ru/locale/ru/lightning/lightning.properties
index a0d6de7..9565c99 100644
--- a/ru/locale/ru/lightning/lightning.properties
+++ b/ru/locale/ru/lightning/lightning.properties
@@ -38,12 +38,12 @@
 # ***** END LICENSE BLOCK *****
 
 # addon description:
-extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Lightning
+extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Iceowl
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.description=Встраиваемый календарь для Thunderbird
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.creator=Mozilla Calendar Project
 
 # Lightning branding
-brandShortName=Lightning
+brandShortName=Iceowl
 
 # Task mode title
 taskModeApplicationTitle=Задачи
@@ -72,7 +72,7 @@ imipBarRequestText=Это письмо содержит приглашение 
 imipBarUpdateText=Это письмо содержит обновление существующего события.
 imipBarAlreadyProcessedText=Это письмо содержит событие, которое уже обработано.
 imipBarReplyText=Это письмо содержит ответ на приглашение..
-imipBarUnsupportedText=Это письмо содержит событие, которое эта версия Lightning не может обработать.
+imipBarUnsupportedText=Это письмо содержит событие, которое эта версия Iceowl не может обработать.
 imipBarProcessingFailed=Ошибка обработки письма. Состояние: %1$S.
 imipAcceptInvitation.label=Принять
 imipCancelInvitation.label=Удалить
